### DISCUSSION TITLE
Adding online group with no address, is erroring out.

### DISCUSSION THREAD

**Discussion by WealthySoul** [code4recovery/12-step-meeting-list#1303] [Q&A] [answered]
A39 is now registering online groups. When adding the group with no physical address, the program is erroring out. I did not want to make up addresses. Since this will change with POC or GSR's. Area 39 is considering opening an online district for these 6-7 groups. I believe more areas will be doing this since 71st Panel voted in to let these online groups choose their area. I saw a post about OIAA, which is where some of these groups are coming from. So if this program is used there, why the glitch. 

It was suggested by another area with an online district to use a virtual address to make it work. I was not wanting to ask these groups to do this, but I may. Is it the right thing to ask these groups to do? Or is there a fix here, that I don't know about? I do appreciate your help. - April

![Joy of Living - error  2](https://github.com/code4recovery/online-meeting-list/assets/139665422/6d1ed3a9-b967-4d71-9560-f61e61a2e1ce)
email: wealthysoul7@gmail.com 
mobile: 310-766-7884

**Comment by joshreisner** [comment]
hi april, yes, online groups with no physical address will need an approximate one, `Liberty, MO, USA` should work fine. the error you're seeing above is because "Can I currently attend this meeting in-person" is set to Yes. Set it to No and the error should go away.

**Reply by WealthySoul** [reply]
@joshreisner  
I just tried it. I am still having problems with it saving. Moved setting to "No" put in the approximate Liberty, MO, USA. The save Draft is now frozen. Sorry I tried this last night and didn't mention it. When the "Yes" is on the error sign goes on. With the "No" the Draft button is frozen.

**Reply by joshreisner** [reply, marked as answer]
hm! could you also check that the URL field under Online Meeting Details has a valid conference url in it?

when i try it on our demo site it's ok for me:

![Screenshot 2024-01-12 at 08-33-13 Add New Post ‹ Code for Recovery — WordPress](https://github.com/code4recovery/12-step-meeting-list/assets/1551689/10afd1a8-f44c-4ed8-a0df-5b98c66d6d81)

**Comment by joshreisner** [comment]
You're using 12 Step Meeting List, not Online Meeting List like OIAA uses, I'm transferring this issue.

**Reply by joshreisner** [reply]
Ok it's transferred! We ask that users with issues around the WordPress plugin please add new Discussions, you can also follow the link on the WOrdPress admin page. https://github.com/code4recovery/12-step-meeting-list/discussions Thanks

**Comment by WealthySoul** [comment]
Also with No, general location. I then added Group Notes and an email. The Draft was unfrozen then I recieved this error.
![Joy of Living - error  4](https://github.com/code4recovery/12-step-meeting-list/assets/139665422/66815a77-33c5-4b41-9758-8459e0f417e3)

**Reply by WealthySoul** [reply]
@joshreisner 
I tried a new online group; in case I had bad information.
I saved as a draft at each data box. It did save at the location (yeah!), but the last box, "Contact Information"  box, after filling it out, received the same error at above. But I backed out of that page. Went to meeting dashboard refreshed, then went to edit same meeting. All the data was saved. I checked the save Draft again, it worked. Then Published it worked. 

So I am going to just save draft per box, and I emailed the group to get a new url. I think you where correct that was my problem. 
Thx you!
